### PR TITLE
UI redesign part5

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

**Review by nop33 — COMMENTED** [review]
The header seems wrong in this screen

~~image~~

EDIT: Created threads...

File: apps/mobile-wallet/package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "big-integer": "^1.6.51",
```
Comment (nop33) [inline apps/mobile-wallet/package.json:48]: (Unrelated to this line, just making a thread)

1. The width of the input changes while typing. Is it intended?
2. Button to continue is missing

image

File: apps/mobile-wallet/package.json [hunk 2]
```diff
@@ -48,7 +48,7 @@
...
     "colord": "^2.9.3",
```
Comment (nop33) [inline apps/mobile-wallet/package.json:49]: Some headers are wrong:
1. Import wallet screen
2. Addresses screen

image


image

File: apps/mobile-wallet/package.json [hunk 3]
```diff
@@ -48,7 +48,7 @@
...
     "dayjs": "^1.10.7",
```
Comment (nop33) [inline apps/mobile-wallet/package.json:50]: App crashed when importing a wallet...

image

File: apps/mobile-wallet/ios/Podfile.lock [hunk 4]
```diff
@@ -1618,7 +1618,7 @@ PODS:
     - React-Core
```
Comment (nop33) [inline apps/mobile-wallet/ios/Podfile.lock:1618]: Some paddings are wrong, maybe due to rebase...

image

image

File: apps/mobile-wallet/ios/Podfile.lock [hunk 5]
```diff
@@ -1618,7 +1618,7 @@ PODS:
...
   - RNFlashList (1.6.4):
```
Comment (nop33) [inline apps/mobile-wallet/ios/Podfile.lock:1619]: Some screens are missing a way to go back. The button is there but hidden. I can click top left and go back:
1. Receive screen, step 1
2. Send screen, step 1

image
image

File: apps/mobile-wallet/ios/Podfile.lock [hunk 6]
```diff
@@ -1618,7 +1618,7 @@ PODS:
...
     - React-Core
```
Comment (nop33) [inline apps/mobile-wallet/ios/Podfile.lock:1620]: "+" button seems cuttoff:

image

File: apps/mobile-wallet/ios/Podfile.lock [hunk 7]
```diff
@@ -1618,7 +1618,7 @@ PODS:
...
-  - RNGestureHandler (2.16.2):
+  - RNGestureHandler (2.21.2):
     - DoubleConversion
```
Comment (nop33) [inline apps/mobile-wallet/ios/Podfile.lock:1622]: Contact address "dot" and internal address "dot" have diff sizes

image

File: apps/mobile-wallet/ios/Podfile.lock (hunk 4 above)
Comment (mvaivre) [reply apps/mobile-wallet/ios/Podfile.lock:1618]: ah yep, used to be correct. Will fix

File: apps/mobile-wallet/ios/Podfile.lock (hunk 5 above)
Comment (mvaivre) [reply apps/mobile-wallet/ios/Podfile.lock:1619]: mhhh used to work too, strange

File: apps/mobile-wallet/ios/Podfile.lock [hunk 8]
```diff
@@ -1618,7 +1618,7 @@ PODS:
...
     - glog
```
Comment (nop33) [inline apps/mobile-wallet/ios/Podfile.lock:1623]: small gitch on Android with the bottom padding:

image

File: apps/mobile-wallet/ios/Podfile.lock [hunk 9]
```diff
@@ -1618,7 +1618,7 @@ PODS:
...
     - hermes-engine
```
[the remark this answers is not in the document]
Comment (nop33) [reply apps/mobile-wallet/ios/Podfile.lock:1624]: Related to this:

image

File: apps/mobile-wallet/ios/Podfile.lock [hunk 10]
```diff
@@ -2164,7 +2164,7 @@ SPEC CHECKSUMS:
   ReactCommon: f7da14a8827b72704169a48c929bcde802698361
```
Comment (nop33) [inline apps/mobile-wallet/ios/Podfile.lock:2164]: Contact screen is broken

image

File: apps/mobile-wallet/src/components/TransactionListItem.tsx [hunk 11]
```diff
@@ -57,27 +57,11 @@ const TransactionListItem = memo(({ tx, showInternalInflows = false, ...props }:
         </TransactionIcon>
       }
       rightSideContent={
-        <>
-          <AssetLogos>
-            {assets.map((asset) => (
-              <AssetLogo assetId={asset.id} key={asset.id} size={20} />
-            ))}
-          </AssetLogos>
```
Comment (nop33) [inline apps/mobile-wallet/src/components/TransactionListItem.tsx:65]: This means that unknown tokens are now hidden? That can be confusing when sending unknown tokens cause you'll only see some dust alph in the tx list item...

File: apps/mobile-wallet/package.json (hunk 1 above)
Comment (nop33) [reply apps/mobile-wallet/package.json:48]: @mvaivre is this intended or should it be fixed? I can work on it.

File: apps/mobile-wallet/ios/Podfile.lock (hunk 8 above)
Comment (nop33) [reply apps/mobile-wallet/ios/Podfile.lock:1623]: This issue is fixed in https://github.com/alephium/alephium-frontend/pull/1032

Closing

File: apps/mobile-wallet/package.json (hunk 2 above)
Comment (nop33) [reply apps/mobile-wallet/package.json:49]: First is fixed in this PR: https://github.com/alephium/alephium-frontend/pull/1032
Second is already merged. 

Resolving.

File: apps/mobile-wallet/ios/Podfile.lock (hunk 6 above)
Comment (nop33) [reply apps/mobile-wallet/ios/Podfile.lock:1620]: The button was removed all together. Closing.

File: apps/mobile-wallet/package.json [hunk 12]
```diff
@@ -88,7 +88,7 @@
     "react-native-aes-crypto": "^2.1.0",
     "react-native-background-actions": "^3.0.1",
     "react-native-background-timer": "^2.4.1",
-    "react-native-gesture-handler": "^2.16.2",
+    "react-native-gesture-handler": "^2.20.2",
     "react-native-get-random-values": "~1.11.0",
     "react-native-localize": "^3.2.0",
     "react-native-modalize": "^2.1.1",
```
Comment (nop33) [inline apps/mobile-wallet/package.json:94]: There are differences on inset calculations between platforms. Buttons are too low on Android.

image

File: apps/mobile-wallet/package.json (hunk 3 above)
Comment (nop33) [reply apps/mobile-wallet/package.json:50]: It happens consistently, I tried again...

image

File: apps/mobile-wallet/ios/Podfile.lock [hunk 13]
```diff
@@ -2164,7 +2164,7 @@ SPEC CHECKSUMS:
...
   RNCAsyncStorage: 826b603ae9c0f88b5ac4e956801f755109fa4d5c
   RNFlashList: b521ebdd7f9352673817f1d98e8bdc0c8cf8545b
-  RNGestureHandler: 2282cfbcf86c360d29f44ace393203afd5c6cff7
+  RNGestureHandler: 6fee3422fd8c81c5ee756fa72e3d1780e9943d9d
   RNLocalize: b77875884750cb6a58cd6865863fe2ba2729b72b
   RNReanimated: 33c6135cdf421b795c2f1173075a11cca2a39bf2
   RNScreens: b32a9ff15bea7fcdbe5dff6477bc503f792b1208
```
Comment (nop33) [inline apps/mobile-wallet/ios/Podfile.lock:2170]: Toast message doesn't have a background 

image

File: apps/mobile-wallet/ios/Podfile.lock [hunk 14]
```diff
@@ -2164,7 +2164,7 @@ SPEC CHECKSUMS:
   ReactCommon: f7da14a8827b72704169a48c929bcde802698361
   RNCAsyncStorage: 826b603ae9c0f88b5ac4e956801f755109fa4d5c
   RNFlashList: b521ebdd7f9352673817f1d98e8bdc0c8cf8545b
-  RNGestureHandler: 2282cfbcf86c360d29f44ace393203afd5c6cff7
+  RNGestureHandler: 6fee3422fd8c81c5ee756fa72e3d1780e9943d9d
   RNLocalize: b77875884750cb6a58cd6865863fe2ba2729b72b
```
Comment (nop33) [inline apps/mobile-wallet/ios/Podfile.lock:2168]: When something is copied to clipboard, Android renders a "paste" button, but it doesn't work well

image

File: apps/mobile-wallet/ios/Podfile.lock (hunk 7 above)
Comment (nop33) [reply apps/mobile-wallet/ios/Podfile.lock:1622]: Solved in https://github.com/alephium/alephium-frontend/pull/1032

File: apps/mobile-wallet/locales/en-US/translation.json [hunk 15]
```diff
@@ -10,7 +10,6 @@
   "No transactions yet": "No transactions yet",
```
Comment (nop33) [inline apps/mobile-wallet/locales/en-US/translation.json:10]: Not a bug, just not very pretty...

image

File: apps/mobile-wallet/ios/Podfile.lock (hunk 14 above)
Comment (mvaivre) [reply apps/mobile-wallet/ios/Podfile.lock:2168]: Fixed

File: apps/mobile-wallet/ios/Podfile.lock (hunk 13 above)
Comment (mvaivre) [reply apps/mobile-wallet/ios/Podfile.lock:2170]: Fixed

File: apps/mobile-wallet/package.json (hunk 3 above)
Comment (mvaivre) [reply apps/mobile-wallet/package.json:50]: Yeah I have it too. I wonder what made this crash, probably a deps update.

File: apps/mobile-wallet/package.json (hunk 1 above)
Comment (mvaivre) [reply apps/mobile-wallet/package.json:48]: I'll fix it

File: apps/mobile-wallet/ios/Podfile.lock (hunk 14 above)
Comment (nop33) [reply apps/mobile-wallet/ios/Podfile.lock:2168]: styles look good!

I just checked it and it gives me an error when I click "Paste" and then "Continue"

image

Comment (nop33) [reply apps/mobile-wallet/ios/Podfile.lock:2168]: Fixed in:
- https://github.com/alephium/alephium-frontend/pull/1033

File: apps/mobile-wallet/ios/Podfile.lock (hunk 13 above)
Comment (nop33) [reply apps/mobile-wallet/ios/Podfile.lock:2170]: Found another place where it's transparent, adding an asset in send flow

image

File: apps/mobile-wallet/ios/Podfile.lock (hunk 14 above)
Comment (mvaivre) [reply apps/mobile-wallet/ios/Podfile.lock:2168]: Thanks a lot! Merged and rebased!

File: apps/mobile-wallet/locales/en-US/translation.json (hunk 15 above)
Comment (nop33) [reply apps/mobile-wallet/locales/en-US/translation.json:10]: Seems to be fixed

**Pull request merged**